docs/using-services.md

```diff
@@ -0,0 +1,672 @@
+---
```

Comment:
Depending on if we stick with `stacks-guide.md`, this should likely also be named `services-guide.md`

Reply:
fixed